.github/workflows/update-test-ubuntu-git.yml

```diff
@@ -37,20 +37,19 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      # Use `docker/metadata-action` to preserve tags and labels that exist on the GHCR.io container image. 
-      # - name: Extract metadata (tags, labels) for Docker
-      #   id: meta
-      #   uses: docker/metadata-action@v5.5.1
-      #   with:
-      #     images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+      - name: Format Timestamp
+        id: timestamp
+        # Use `date` with a custom format to achieve the key=value format GITHUB_OUTPUT expects.
+        run: date -u "+now=%Y%m%d.%H%M%S.%3NZ" >> "$GITHUB_OUTPUT"
 
       # Use `docker/build-push-action` to build (and optionally publish) the image. 
       - name: Build and push Docker image
         uses: docker/build-push-action@v5.1.0
         with:
           context: .
           file: images/test-ubuntu-git.Dockerfile
-          push: ${{ inputs.publish }}
-          tags:  ${{ env.IMAGE_NAME }}:sha-${{ env.GITHUB_SHA }}
-          # tags: ${{ steps.meta.outputs.tags }}
-          # labels: ${{ steps.meta.outputs.labels }}
+          # For now, attempts to push to ghcr.io must target the `main` branch.
+          # In the future, consider also allowing attempts from `releases/*` branches.
+          push: ${{ inputs.publish && github.ref_name == 'main' }}
```

Comment:
Also pushes `:latest` right?

Reply:
No.  I haven't included a `:latest` tag in this implementation.

There's only one known use case (`test-proxy`), so I'm OK to pinning that use case to an explicit timestamp.